stub/src/test/java/io/grpc/stub/CallsTest.java

```diff
@@ -69,6 +69,7 @@
   @Test public void unaryFutureCallSuccess() throws Exception {
     Integer req = 2;
     ListenableFuture<String> future = Calls.unaryFutureCall(call, req);
+    @SuppressWarnings("rawtypes")
     ArgumentCaptor<Call.Listener> listenerCaptor = ArgumentCaptor.forClass(Call.Listener.class);
```

Comment:
The trick with ArgumentCaptor is to pass null to forClass(), and then the generics are happy and ArgumentCaptor ignores the class. :-)

That doesn't work with mock() since it actually uses the class, but it does here.